src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -51,5 +76,175 @@ public void InvalidSelectionChangeInvalidPrevious()
 			Assert.IsNull(list.SelectedItem);
 #endif
 		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_ContainerSet_Then_ContentShouldBeSet()
+		{
+			var resources = new TestsResources();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'resources'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5961910)